bin/console

```diff
@@ -7,8 +7,10 @@ require 'http_logger'
 HttpLogger.log_headers  = true
 HttpLogger.logger       = Logger.new STDOUT
 
-
 require 'pry'
 Pry.pager = nil
 
-Pry.start
+# DO NOT COMMIT YOUR API KEY
+API_KEY = 'xxx'
```

Comment:
This is way better than the file I have on my desktop, what if we use something like Figaro and load in a `development-secrets.yml`? That way we can `.gitignore` and I won't commit this